assessments/projects/custodian/joint-review.md

```diff
@@ -0,0 +1,556 @@
+or DoS of a deployment by an internal or external actor - thus giving the user a false sense of security when c7n is disabled. A tertiary concern is the
+provisioning of too permissive a policy within the cloud environment for c7n; this would make it an attrative target for lateral movement and other activities
+for an attacker who had gained access via some other means. Finally, comprehensive support for Kubernetes is not currently available; the review team would like to see more
+active support for Kubernetes and other CNCF projects as c7n matures.
+
+A “hands-on” penetration test was not performed, nor was a comprehensive code review, code fuzzing, or code audit.  The review team strongly recommends pursuing this as a 
+follow up with appropriate CNCF support and guidance.
+
+## Table of Contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Joint-review use](#joint-review-use)
+* [Intended use](#intended-use)
+* [Project design](#project-design)
+  * [Functions and features](#functions-and-features)
+    * [Security functions and features](#security-functions-and-features)
+* [Configuration and set-up](#configuration-and-set-up)
+* [Project compliance](#project-compliance)
+* [Security analysis](#security-analysis)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+  * [Closed security issues and vulnerabilities](#closed-security-issues-and-vulnerabilities)
+* [Roadmap](#roadmap)
+* [Appendix](#appendix)
+
+## Metadata
+
+| Project Home                    | https://github.com/cloud-custodian                                                                               |
+|---------------------------------|------------------------------------------------------------------------------------------------------------------|
+| Security Provider               | Yes, however many users implement Cloud Custodian for non security purposes, i.e., cost optimization, governance |
+| Languages                       | Python, YAML |
+| Incubation PR                   | https://github.com/cncf/toc/pull/480 |
+| SIG-Security Assessment Request | https://github.com/cncf/sig-security/issues/307 |
+| Self-Assessment Document        | https://docs.google.com/document/d/1s88ifDtFJzGX1O_ve6HM6_vbTkpsWDKWRDuxt9RHdcs/ |
+| SBOM                            | [poetry configuration in pyproject.toml](https://github.com/cloud-custodian/cloud-custodian/blob/master/pyproject.toml), [poetry lock file](https://github.com/cloud-custodian/cloud-custodian/blob/master/poetry.lock), [pip requirements file](https://github.com/cloud-custodian/cloud-custodian/blob/master/requirements.txt), [release process](https://cloudcustodian.io/docs/developer/packaging.html) |
+
```

Comment:
Can we add list of reviewers / project participants for the joint review (we should fix the template to include this)

Reply:
I'm thinking a copy of our initial conflict-of-interest disclosures (from the GH issue/ticket) could also be included verbatim in some appendix at the end.

Reply:
>I'm thinking a copy of our initial conflict-of-interest disclosures (from the GH issue/ticket) could also be included verbatim in some appendix at the end.

it probably is not necessary , it just needs a chair sign off on the issue. it should all be saved in Github history

Reply:
Sounds good.

Reply:
I have an open PR #787  with the correct table to add in the reviewers.

Reply:
I think this is resolved now...please review